### PR TITLE
v5 - add support for PHPUnit v10

### PR DESCRIPTION
**Pull request by freekmurze**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
-.php-cs-fixer.cache
 .phpunit.result.cache
-build
 composer.lock
 coverage
 docs
 phpunit.xml
 psalm.xml
 vendor
+ .phpunit.cache
```
Comment (jdreesen) [inline .gitignore:8]:
```suggestion
.phpunit.cache
```

File: .phpunit.cache/test-results [hunk 2]
```diff
@@ -0,0 +1 @@
...Test::it_can_serialize_a_simple_object":0,"Spatie\\Snapshots\\Test\\Unit\\Drivers\\ObjectDriverTest::it_can_serialize_a_class_instance":0.001,"Spatie\\Snapshots\\Test\\Unit\\Drivers\\TextDriverTest::it_can_serialize_laravel_route_list":0,"Spatie\\Snapshots\\Test\\Unit\\Drivers\\TextDriverTest::it_can_serialize_when_given_OS_dependant_line_endings":0,"Spatie\\Snapshots\\Test\\Unit\\Drivers\\XmlDriverTest::it_can_serialize_a_xml_string_to_pretty_xml":0,"Spatie\\Snapshots\\Test\\Unit\\Drivers\\XmlDriverTest::it_can_only_serialize_strings":0,"Spatie\\Snapshots\\Test\\Unit\\Drivers\\YamlDriverTest::it_can_serialize_a_yaml_string":0,"Spatie\\Snapshots\\Test\\Unit\\Drivers\\YamlDriverTest::it_can_serialize_a_yaml_array":0,"Spatie\\Snapshots\\Test\\Unit\\FilenameTest::it_creates_a_filename_which_is_valid_on_all_systems#0":0,"Spatie\\Snapshots\\Test\\Unit\\FilenameTest::it_creates_a_filename_which_is_valid_on_all_systems#1":0,"Spatie\\Snapshots\\Test\\Unit\\SnapshotTest::it_has_an_id":0,"Spatie\\Snapshots\\Test\\Unit\\SnapshotTest::it_has_a_filename_based_on_its_id_and_its_drivers_extension":0,"Spatie\\Snapshots\\Test\\Unit\\SnapshotTest::it_has_a_filename_which_is_valid_on_all_systems":0}}
```
Comment (jdreesen) [inline .phpunit.cache/test-results:1]: This file should not be in git.

**Comment by dazz** [comment]
Hey, can we get a tag with those changes?

**Comment by dazz** [comment]
Thank you 🐘🖖

**Pull request merged**